--- x86-mingw32-build.sh.conf
+++ x86-mingw32-build.sh.conf	2012-02-09 16:27:33.642305221 +0100
@@ -43,16 +43,16 @@
 #    match your own host configuration; note that you MUST have WRITE
 #    permission for each directory specified.
 #
-     assume WORKING_DIR                   $HOME/tmp/mingw-3.4.5
-     assume PACKAGE_DIR                   $HOME/packages/mingw-3.4.5
-     assume INSTALL_DIR                   $HOME/mingw32
+     assume WORKING_DIR                   `pwd`/../../../build/mingw
+     assume PACKAGE_DIR                   `pwd`/../packages
+     assume INSTALL_DIR                   `pwd`/../../../install
 #
 # 4) Package versions, from which the compiler will be built.
 #
-     assume GCC_VERSION                   3.4.5-20060117-2
-     assume BINUTILS_VERSION              2.20.1
-     assume RUNTIME_VERSION               3.18
-     assume W32API_VERSION                3.14
+     assume GCC_VERSION                   4.6.2-1
+     assume BINUTILS_VERSION              2.22
+     assume RUNTIME_VERSION               3.20
+     assume W32API_VERSION                3.17
 #
 # 5) Automatic patching of downloaded packages: if any appropriately
 #    named patch sets are placed in the nominated patches directory,
@@ -93,11 +93,11 @@
 #    the corresponding package is available in the PACKAGE_DIR, or
 #    if it is selected for download.
 #
-     option GCC_LANGUAGE_OPTIONS          ada
+#    option GCC_LANGUAGE_OPTIONS          ada
      option GCC_LANGUAGE_OPTIONS          c++
-     option GCC_LANGUAGE_OPTIONS          f77
-     option GCC_LANGUAGE_OPTIONS          java
-     option GCC_LANGUAGE_OPTIONS          objc
+#    option GCC_LANGUAGE_OPTIONS          f77
+#    option GCC_LANGUAGE_OPTIONS          java
+#    option GCC_LANGUAGE_OPTIONS          objc
 #
 # 8) User preferences for configuration options.  These have been
 #    preconfigured to match the setup of the prebuilt compiler suite,
